sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -5,11 +5,10 @@
 ### Features Added
 - Includes all features from 12.26.0-beta.1
 
-## 12.25.0 (2025-07-14)
+## 12.25.1 (2025-09-24)
```

Comment:
I think we need a new entry for 12.25.1, instead of replacing the entry for 12.25.0